pkg/ocm/ocm.go

```diff
@@ -188,8 +193,12 @@ func New(ocmcOcmUrl string) (*Config, error) {
 
 // url takes a string in the form of urlAliases, and returns
 // the actual OCM URL
-func url(s string) string {
-	return urlAliases[s]
+func url(s string) (string, error) {
+	u, ok := urlAliases[s]
+	if !ok {
+		return "", errInvalidOcmUrl
```

Comment:
While this returns urlAliases, should we also exit at the beginning of this function if the url starts with `http` to allow users to point to a non-aliased OCM URL (or something running locally, etc)? Just trying to think of additional use-cases outside of our specific one that we can "future-proof" for little up-front work :D